src/AddTodoForm.jsx

```diff
@@ -0,0 +1,19 @@
+function AddTodoForm({ onAddTodo }) {
+  const handleAddTodo = event => {
+    event.preventDefault();
+    const todoTitle = event.target.querySelector('input[name="title"]').value;
```

Comment:
Is not a good practice to use query selectors


> Can You Use QuerySelector in React?
> 
> If you're working with React, you're probably familiar with the concept of a "virtual DOM" - a JavaScript representation of the actual DOM. This is what React uses to determine what changes need to be made to the actual DOM.
> 
> One of the benefits of the virtual DOM is that it allows React to optimize updates to the UI. When React is working with the virtual DOM, it can batch together multiple changes and then apply them all at once to the actual DOM. This is more efficient than making individual changes to the DOM, and it helps to avoid potential issues with React components getting out of sync with the actual DOM.
> 
> When you try to modify the element that was retrieved using querySelector, you're no longer taking the advantage of React virtual DOM. It's still possible to use querySelector in React, however, React encourages developers not to use it. Instead, we should aim to use refs if possible.
> REF: https://www.upbeatcode.com/react/how-to-use-queryselector-in-react/